src/fastcs/transport/epics/pva/_pv_handlers.py

```diff
@@ -61,6 +63,7 @@ async def _run_command(self) -> dict:
         try:
             await self._command()
         except Exception as e:
+            print(traceback.format_exc())
```

Comment:
I am confused why this is specifically in the p4p transport. Are commands failing already visible for all other transports or just ca? It would be good if the transport didn't have to implement anything for failed commands to be visible to the terminal.